receiver/otlpreceiver/otlp_test.go

```diff
@@ -29,9 +30,11 @@ import (
 	"github.com/gogo/protobuf/proto"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	spb "google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	gproto "google.golang.org/protobuf/proto"
```

Comment:
Nit: I think we usually use import alias `gogoproto` for Gogo lib and just `proto` for Google protobuf lib.

Reply:
Done.